lib/unleash/strategy/user_created_after.rb

```diff
@@ -15,9 +15,12 @@ def is_enabled?(params = {}, context = nil)
         return false unless context.instance_of?(Unleash::Context)
         return false if forbidden_org_uuids(params).include?(current_org_uuid(context))
 
+        user_creation_date = user_created_at(context)
++       return false if user_creation_date.nil?
```

Comment:
What if, instead of parsing nil, we're parsing a non-parseable datetime, like `non-parseable`, `1` , etc? Should we cover this scenario as well?

Reply:
Sorry for the delay 😅 good point! I converted the date to a string so it will always be a valid object to parse, but, if the value is not appropriate, it will fall in the `rescue ArgumentError`. I don't know if it is the best solution but I think it does the trick 😬  WDYT @gbrlcustodio ?

Reply:
Welp, it doesn't seem to be enough for covering mentioned scenarios, whenever I try parsing inputs like `''` and `'1'`, I get a `Date::Error` (see in the examples below). Do we have any testing scenarios covering those?

```ruby
irb(main):005:0> DateTime.parse('1')
Traceback (most recent call last):
        ...
        1: from (irb):5:in `parse'
Date::Error (invalid date)
irb(main):006:0> DateTime.parse('')
Traceback (most recent call last):
        ...
        1: from (irb):6:in `parse'
Date::Error (invalid date)
```

Am I doing something wrong?

Reply:
So, the `Date::Error` inherits from `ArgumentError` (which is the one being rescued) so these scenarios are being covered 😸

Reply:
That explains! Thanks for letting me know :1st_place_medal: